src/error-snippets/index.ts

```diff
@@ -0,0 +1,54 @@
+import { codeFrameColumns } from "@babel/code-frame";
+import { findRelevantStackFrame, resolveWithStackFrame } from "./frames";
+import { extractSourceMaps, resolveWithSourceMap } from "./source-maps";
+import { getSourceCodeFile, formatFileNameHeader } from "./utils";
+import logger from "../utils/logger";
+import { SNIPPET_LINES_ABOVE, SNIPPET_LINES_BELOW } from "./constants";
+import type { ResolvedFrame, SufficientStackFrame, WithSnippetError } from "./types";
+
+const stackFrameResolver = async (stackFrame: SufficientStackFrame): Promise<ResolvedFrame> => {
+    const fileContents = await getSourceCodeFile(stackFrame.fileName);
+    const sourceMaps = await extractSourceMaps(fileContents, stackFrame.fileName);
+
+    return sourceMaps ? resolveWithSourceMap(stackFrame, sourceMaps) : resolveWithStackFrame(stackFrame, fileContents);
+};
```

Comment:
`getSourceCodeFile` extracts source code from file and applies source maps, if it is in http headers: https://docs.google.com/document/d/1U1RGAehQwRypUTovF1KRlpiOFze0b-_2gc6fAH0KY0k/edit#heading=h.lmz475t4mvbx

So in other places we dont have to think "is the file from node:fs or browser:fetch?"

Reply:
source maps will be evaluated, if exists